src/main/java/core/basesyntax/service/operation/BalanceOperation.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service.operation;
+
+public class BalanceOperation implements OperationHandler {
```

Comment:
The class name `BalanceOperation` suggests that it handles the balance operation, but the method name `getQuantity` is not very descriptive of what the operation actually does. Consider renaming the method to better reflect its purpose, such as `applyBalanceToQuantity`.